enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,359 @@
+---
+title: set_eip_nlb_ingress
+authors:
+  - miheer
+reviewers:
+  - Miciah
+  - frobware
+  - gspence
+  - candace
+approvers:
+  - Miciah
+  - frobware
+  - gspence
+api-approvers:
+  - joel
+  - deads
+creation-date: 2024-03-06
+last-updated: 2024-03-22
+tracking-link:
+  - "https://issues.redhat.com/browse/NE-1274"
+see-also:
+  - ""
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# Set AWS EIP For NLB Ingress Controller
+
+## Summary
+
+This enhancement allow user to set AWS EIP for the NLB default or custom ingress controller.
+This is a feature request to enhance the IngressController API to be able to support static IPs during
+- Install time
+- Custom NLB ingress controller creation 
+- Reconfiguration of the router.
+
+## Motivation
+
+### User Stories
+
+- As an administrator, I want to provision EIPs and use them with an NLB IngressController.
+- As a user or admin, I want to ensure EIPs are used with NLB on default router at install time.
+- As a user, I want to reconfigure default router to use EIPs.
+- As a user of OpenShift on AWS (ROSA), I want to use static IPs (and therefore AWS Elastic IPs) so that 
+  I can configure appropriate firewall rules.
+  I want the default AWS Load Balancer that they use (NLB) for their router to use these EIPs.
+
+### Goals
+- Users are able to use EIP for a NLB Ingress Controller.
+- Users are able to create a NLB default Ingress Controller during install time with the EIPs specified during install.
+- Any existing ingress controller should be able to be reconfigured to use EIPs. 
```

Comment:
As mentioned earlier, "reconfigure" should really be "update with restart", right?

Reply:
yes

Reply:
I see that https://github.com/openshift/enhancements/compare/1137a8acba5a4c72ae496c34fa93b97dd18d3990..79d098963268d49c7e4b7d64ffe1e25878c2b6c0 implemented this suggestion.  

To me, "restart" implies that the router pod has to be restarted.  How about this?
```markdown
- The ELB of an existing IngressController should be able to be reconfigured to use specific EIPs by updating the IngressController CR and recreating the ELB. 
```

Reply:
added with slight changes.